contracts/DegenLockToken.sol

```diff
@@ -0,0 +1,197 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.20;
```

Comment:
The version pragma can be set to `^0.8.0` in all the contracts and the compiler version can be defined in the Hardhat config: https://github.com/degen-token/degen-smart-contracts/blob/main/hardhat.config.ts#L45.

Reply:
Fixed.